go/Makefile

```diff
@@ -0,0 +1,29 @@
+install-tools:
+	@cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
+build: build-glide-core build-glide-client generate-protobuf
+	go build ./...
+
+build-glide-core:
+	cd ../glide-core; cargo build --release
+
+build-glide-client:
+	cargo build --release
+
+generate-protobuf:
+	mkdir -p protobuf
+	protoc --proto_path=../glide-core/src/protobuf \
+		--go_opt=Mconnection_request.proto=github.com/aws/glide-for-redis/go/protobuf \
+		--go_opt=Mredis_request.proto=github.com/aws/glide-for-redis/go/protobuf \
+		--go_opt=Mresponse.proto=github.com/aws/glide-for-redis/go/protobuf \
+		--go_out=./protobuf \
+		--go_opt=paths=source_relative \
+		../glide-core/src/protobuf/*.proto
+
+lint:
```

Comment:
you can call call this from CI

Reply:
Sure, I currently have CI running go vet and staticcheck as separate steps in case that would make the logs clearer, but maybe it would be better to group them together and just call this target like you're suggesting. I'll make the change